docs/content/tutorial/assets/defining-an-asset.mdx

```diff
@@ -56,17 +56,17 @@ Serving dagit on http://127.0.0.1:3000 in process 70635
 
 You should be able to navigate to <http://127.0.0.1:3000> in your web browser and view your asset.
 
-<img src="/images/dagster/guides/asset-tutorial/defining-an-asset/defining_an_asset.png" />
+<img src="/images/guides/asset-tutorial/defining_an_asset.png" />
 
 Clicking on the "Materialize All" button will launch a run that will materialize the asset. After that run has completed, the shaded box underneath "cereals" holds information about that run. Clicking on the Run ID, which is the string of characters in the upper right of that box, will take you to a view that includes a structured stream of logs and events that occurred during its execution.
```

Comment:
This text should now be revised to match the new UI. Eg

Click on the "Materialize" button launches a run that materializes the asset. After the run has completed the Activity tab on the assets page holds information about that run. Click on the Run ID in the activity tab to go see details about the run. This opens a view that includes a structured stream of logs and events that occurred during the run execution.